### ISSUE TITLE
Proposal: visualize useful synthetic information when user clicks 'Tags' 

### ISSUE THREAD

**Issue by Spartachetto** [nextcloud/server#2143]
When user clicks 'Tags', s\he opens a view which is remarkable for its lack of information about tags.

That place could be used to display some summary of the tags used which could be useful or interesting for most of the users.

Here is a modest proposal:

--------------------------
you are using **58** tags on **4678** objects.

The tagged objects have **2** tags on average.

The five most used tags are:

- **one** (_704 times_)
- **two** (_578 times_)
- **three** (_550 time_)
- **four** (_489 times_)
- **five** (_265 times_)

You are tagging mainly files (_3921 objects_), but you tag also contacts (_400 objects_), and calendar items (_357 objects_)

--------------------------

The last phrase is an extension that considers the possibility of an unified tag system ( ! )

The text tries to provide a lot of info in a few lines. It shows
- basic information on how much you are using the tag tool
- the most used tags by you
- basic information on how you are using the tag tool 

In case of interest, there are three main paths to expand this feature:
- display more generic information
- display information about the other users
- display specific information on a tag after the user selects it

**Comment by nickvergessen** [comment]
Thanks, sounds good and added to the overview issue #1465

**Comment by Spartachetto** [comment]
Each time I happen to reread what I wrote, I feel that the last part of what I wrote is not clear.

I'll rephrase it to be more clear

----------------------------------------------------------------------

In case of interest, there are three main paths to expand this feature:

*    display more generic information, meaning increase even more the information provided when you open the tag window  
*    display information about the other users. The descriptive text provides information only about how you (i.e.: the active user) use the tags; it could be interesting to discover how the other users of your instance use the same tags. Of course only for the files you can access. 
*    display specific information on a tag after the user selects it. After you select at least one tag you could be interested to read the same synthetic information restricted to the selected tag(s)

**Comment by DarkknightAK** [comment]
https://robinpowered.com/blog/best-practice-system-for-organizing-and-tagging-github-issues/

What I like here is that you have tag labels and as second item different colors where each color is for an specific group.

For NC the main group could be 
- global labels, 
- user labels, 
- auto labels
Or something like
- sales
- back-office
- support
- ...
Where the main group have sublabels with the same colors.

https://static.robinpowered.com/blog/images/github-tagging-style/robin-github-issue-tag-system.png
